app/controllers/catalog_controller.rb

```diff
@@ -78,7 +78,7 @@ def self.uploaded_field
       qt: "search",
       rows: 10,
       qf: IiifPrint.config.metadata_fields.keys.map { |attribute| "#{attribute}_tesim" }
-                   .join(' ') << "title_tesim description_tesim all_text_timv file_set_text_tsimv",
+                   .join(' ') << " title_tesim description_tesim all_text_timv file_set_text_tsimv",
```

Comment:
Note to future selves: the leading space is important and will cause query errors because title_tesim will then get merged into another word.

Reply:
maybe adding a comment to alert future devs would be helpful here.

Reply:
added!